### PR TITLE
T/token pairing tests

### PR DESCRIPTION
**Pull request by penk85**
Jest test snapshots for: Token Generate Panel, Peer Edit Panel.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ jobs:
       - uses: actions/checkout@v1
       - uses: actions/setup-node@v1
         with:
-          node-version: '12.x'
+          node-version: '12.16'
```
Comment (roosnic1) [inline .github/workflows/test.yml:16]: Please revert that change

File: package.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
     "setup": "run-s --print-label setup:**",
     "dev": "run-p start:docker watch",
     "clean": "run-p --print-label clean:**",
-    "test": "run-p --print-label test:**",
+    "test": "run-s --print-label test:**",
```
Comment (roosnic1) [inline package.json:12]: please revert that change

**Review by roosnic1 — CHANGES_REQUESTED** [review]
looks good

**Review by roosnic1 — APPROVED** [review]

**Review by 0815pascal — APPROVED** [review]

**Review by faebebin — APPROVED** [review]
Looks good. Everything works for me locally too.

**Pull request merged**